docs/2101UserGuide.md

```diff
@@ -216,84 +216,107 @@ Examples:
 <hr class="feature-class-separator">
 
 ### View commands
+In CCACommander, you can quickly change the view to fit your current use case, be it to check for attendance of an event, 
+filtering for a member and more!
 
-#### List all Members and all Events : `list`
+#### View Events of Member : `viewMember`
+
+If you want to view the events in which a particular member is enrolled, you can do so via the `viewMember` command.
+
+Here’s how you can use it:
+`viewMember MEMBER_INDEX`
 
-List all members and all events in the CCA in two separate columns.
+Here is an example command where we want to view the events in which the member at index 3 in our member list is enrolled:
 
-Format: `list`
+`viewMember 3`
 
-A GUI similar to the one below will be shown after entering the command.
-![list](images/list.png)
+<figure>
+    <img src="images/viewMemberPreCommand.png"
+         alt="viewMember Pre Command">
+    <figcaption>Before executing the viewMember command</figcaption>
+</figure>
+<br>
+<figure>
+    <img src="images/viewMemberPostCommand.png"
+         alt="viewMember Post Command">
+    <figcaption>After executing the viewMember command</figcaption>
+</figure>
 
 <hr class="command-separator">
 
-#### View Events of Member : `viewMember`
+#### View Members of Event : `viewEvent`
 
-Lists all the events of a specified member index.
+If you want to view the members which a particular event has, you can do so via the `viewEvent` command.
 
-Format: `viewMember MEMBER_INDEX`
+Here’s how you can use it:
+`viewEvent EVENT_INDEX`
 
-* Views the events of the member at the specified `MEMBER_INDEX`.
-* The index refers to the index number shown in the **currently displayed** member list.
-* The index **must be a positive integer** that is within the range of the length of the member list.
+Here is an example command where we want to view the members that the event at index 1 of our event list has:
 
-Examples:
-* `viewMember 1` displays events of the 1st member in the member list.
-* `viewMember 10` displays events of the 10th member in the member list.
+`viewEvent 1`
 
-A GUI similar to the one below will be shown after entering the command if the member is enrolled in events.
-The hours and remarks for each event is reflected here.
-![viewMember](images/viewMember.png)
+<figure>
+    <img src="images/viewEventPreCommand.png"
+         alt="viewEvent Pre Command">
+    <figcaption>Before executing the viewEvent command</figcaption>
```

Comment:
Should we format this as:
```suggestion
    <figcaption>Before executing the `viewEvent` command</figcaption>
```
Likewise for those below

Reply:
Ok.

Reply:
Oh I just noticed that too, try this instead:
```suggestion
    <figcaption>Before executing the <code>viewEvent</code> command</figcaption>
```

Reply:
Alright!